config.rb

```diff
@@ -5,11 +5,7 @@
 ###
 
 set :markdown_engine, :redcarpet
-set :markdown,
-  fenced_code_blocks: true,
-  smartypants: true,
-  with_toc_data: true,
-  renderer: TechDocsHTMLRenderer
+set :markdown, renderer: TechDocsHTMLRenderer
```

Comment:
Yep, that'd be more explicit if that works, allows markdown config to still be changed in the config file too.